<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UnzipWizard</class>
 <widget class="QWizard" name="UnzipWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wizard</string>
  </property>
  <widget class="QWizardPage" name="wizardPage1">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Let's unzip the resource jar here!</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="UnzipPage" name="wizardPage2">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>First, enter the resource jar here!</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="openBtn">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="document-open"/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>And the destination directory here!</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLineEdit" name="lineEdit_2"/>
      </item>
      <item>
       <widget class="QPushButton" name="openBtn_2">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="document-open"/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Meanwhile, you can reset translation and resource file directory.</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="resetBtn">
      <property name="text">
       <string>Reset Directory</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="UnzipPage" name="wizardPage">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Unzip is in progress......</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage_2">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Please enter the domain of the resource.</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="domainEdit">
      <property name="text">
       <string>minecraft</string>
      </property>
      <property name="placeholderText">
       <string>Domain</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>You can rechoose the directory of resource.</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Translation file</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Not found!</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="transDirBtn">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="document-open"/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Recipes file</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Not found!</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="recipesDirBtn">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="document-open"/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Assets file</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Not found!</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="iconsDirBtn">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="document-open"/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage_3">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>The progress is finished!</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UnzipPage</class>
   <extends>QWizardPage</extends>
   <header>unzippage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
